src/lib/annotations/doc/CreateHighlightDialog.js

```diff
@@ -315,6 +312,39 @@ class CreateHighlightDialog extends EventEmitter {
      * @return {HTMLElement} The element containing Highlight creation UI
      */
     createElement() {
+        const caretTemplate = this.isMobile
```

Comment:
should we move these templates out into constants somewhere?

Reply:
That would've made more sense than what I did :D DONE!